### DISCUSSION TITLE
OneDrive proxy开 下载速度变慢了

### DISCUSSION THREAD

**Discussion by weizai408** [AlistGo/alist#971] [Q&A]
添加的OneDrive，proxy开 使用的vps本身流量，最近发现下载速度慢很多，只能到3MB/S；而下载vps本地硬盘的速度能到30MB/S。
使用其他程序挂载同一OneDrive，也走vps流量，速度正常。
希望看看哪里出问题了。

**Comment by xhofe** [comment]
![image](https://user-images.githubusercontent.com/36558727/164154118-cc4ac441-031a-475c-94c2-2be88c887069.png)

**Reply by weizai408** [reply]
> ![image](https://user-images.githubusercontent.com/36558727/164154118-cc4ac441-031a-475c-94c2-2be88c887069.png)

建议用浏览器直接下载测试，我试了6台vps 都有同样的情况。其他网盘程序正常。

**Reply by xhofe** [reply]
![image](https://user-images.githubusercontent.com/36558727/164166199-3b3acd39-5295-4395-a4c5-72faa4c1f15b.png)

**Reply by weizai408** [reply]
> ![image](https://user-images.githubusercontent.com/36558727/164166199-3b3acd39-5295-4395-a4c5-72faa4c1f15b.png)

1、挂载的OneDrive是否开启了中转
2、你可以对比下vps本地文件的下载速度和中转的OneDrive的下载速度，使用浏览器直接下载。
3、你这个12.2MB/S的速度虽然很快，但我估计vps本地文件的速度更快。

**Reply by xhofe** [reply]
我不想再回了，proxy的代码你可以自己看没有改动过。
